.github/workflows/integration-tests.yml

```diff
@@ -54,7 +54,7 @@ jobs:
   test-windows:
     name: Integration Test on Windows
     needs: test-nix
-    timeout-minutes: 30
+    timeout-minutes: 10
```

Comment:
no reason to change this

Reply:
I'm getting a bit tired of having to wait 30 minutes to see if the 472 run crashes. Have you got any better ideas?

Reply:
it usually crashes before the 30 min is up for me :shrug: 
anyway approved